lib/sentences.js

```diff
@@ -9,7 +9,7 @@ const { prepositions } = require('./prepositions');
 
 const sentences = {
 
-  sentence: [
+  sentence: () => [
```

Comment:
This is a good start. But do we really need to allocate six sentences for every sentence that we pick? I get that it's done to use "chooseRandom", but it seems like a waste of memory